src/v/config/configuration.cc

```diff
@@ -763,6 +764,12 @@ configuration::configuration()
       "Enable SASL authentication for Kafka connections.",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       false)
+  , enable_authorization(
+      *this,
+      "enable_authorization",
+      "Enable authorization for Kafka connections.",
```

Comment:
Maybe a few more words here to explain the effect of unset, true, false?  (I think this is something along the lines of explaining when/if it overrides the per-listener state?)

Reply:
Done